documentation/release_notes.rst

```diff
@@ -8,6 +8,65 @@ The Intel® oneAPI DPC++ Library (oneDPL) accompanies the Intel® oneAPI DPC++/C
 and provides high-productivity APIs aimed to minimize programming efforts of C++ developers
 creating efficient heterogeneous applications.
 
+New in 2022.8.0
+===============
+
+New Features
+------------
+- Added support of host policies for ``histogram`` algorithms.
+- Added support for specification of a size limit for the output range in the range-based ``merge`` algorithm.
```

Comment:
```suggestion
- Added support for a undersized output range in the range-based ``merge`` algorithm.
```
That is based on my understanding of the update - it did not change API for "specifying" any size limit, it changed the implementation to work correctly if the output range is too small to include all elements of the input.